pkg/services/apiserver/options/ready-roundtripper.go

```diff
@@ -0,0 +1,20 @@
+package options
+
+import (
+	"net/http"
+)
+
+// NOTE: both dataplane aggregator and kubernetes aggregator (when enterprise is linked) have logic around
+// setting this RoundTripper as ready, however, kubernetes aggregator skips it is part of the delegate chain
```

Comment:
> kubernetes aggregator skips it is part of the delegate chain

is this comment accurate? if so, where is the skip happening?

Reply:
it's a skip in the sense, that run is skipped. Should i change the language?

Reply:
actually, I made it better, but my change got lost, trying again

Reply:
pushed!